.vscode/launch.json

```diff
@@ -1,6 +1,33 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "Python: Current File",
+      "type": "python",
+      "request": "launch",
+      "program": "${file}",
+      "console": "integratedTerminal",
+      "justMyCode": true
+    },
+    {
+      "name": "Python: pip installed module",
+      "type": "python",
+      "request": "launch",
+      "module": "validator.main",
+      "console": "integratedTerminal",
+      "justMyCode": true
+    },
+    {
+      "name": "Python: validator args",
+      "type": "python",
+      "request": "launch",
+      "module": "validator.main",
+      "args": [
+        "C:\\gis\\Projects\\BroadbandFabric\\working data\\SLCo\\SLCo_Address_Point_Challenge_Final.csv"
```

Comment:
**suggestion**: move this file into the repo

Reply:
That was just in there for debugging purposes, I may just pull that whole section from the file.